docs/DeveloperGuide.md

```diff
@@ -955,6 +955,39 @@ testers are expected to do more *exploratory* testing.
    2. **Test case** (Duplicate house): `addSeller n/Carl Lim p/98765432 e/johncarl@example.com type/Hdb street/Clementi Ave 2 blk/311 level/02 unitNo/25 postal/578578 price/999999999` <br>
       **Expected**: No seller is added. Error indicating that the house already existed in the data. <br>
 
+[//]: # (@@author zengzihui)
+### Adding a buyer
```

Comment:
Change from `Adding a buyer` -> `Adding a Buyer`

Reply:
Updated already

Reply:
This is a very good fix.